rdt/transformers/numerical.py

```diff
@@ -626,3 +626,99 @@ def _reverse_transform(self, data):
+                  value was missing. Then use it to recreate missing values.
+                * ``None``: Do nothing with the missing values on the reverse transform. Simply
+                  pass whatever data we get through.
+        min_value (float):
+            The min value for the logit function. Defaults to 0.
+        max_value (float):
+            max_value (float): The max value for the logit function. Defaults to 1.0.
+        learn_rounding_scheme (bool):
+            Whether or not to learn what place to round to based on the data seen during ``fit``.
+            If ``True``, the data returned by ``reverse_transform`` will be rounded to that place.
+            Defaults to ``False``.
+    """
+
+    def __init__(
+        self,
+        missing_value_replacement='mean',
+        missing_value_generation='random',
+        min_value=0.0,
+        max_value=1.0,
+        learn_rounding_scheme=False,
+    ):
+        super().__init__(
+            missing_value_replacement=missing_value_replacement,
+            missing_value_generation=missing_value_generation,
+            learn_rounding_scheme=learn_rounding_scheme,
+        )
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def _validate_logit_inputs(self, data):
+        out_of_range_vals = data[(data < self.min_value) | (data > self.max_value)]
+        if len(out_of_range_vals) > 0:
+            num_vals_to_print = 5
+            out_of_range_vals = [str(x) for x in sorted(out_of_range_vals, key=lambda x: str(x))]
+            if len(out_of_range_vals) > 5:
+                extra_missing_vals = f'+ {len(out_of_range_vals) - num_vals_to_print} more'
+                out_of_range_vals = (
+                    f'[{", ".join(out_of_range_vals[:num_vals_to_print])} {extra_missing_vals}]'
+                )
+            else:
+                out_of_range_vals = f'[{", ".join(out_of_range_vals)}]'
+
+            raise InvalidDataError(
+                f"Unable to apply logit function to column '{self.columns[0]}' due to out of "
+                f'range values ({out_of_range_vals}).'
+            )
+
+    def _fit(self, data):
+        self._validate_logit_inputs(data)
+        return super()._fit(data)
+
+    def _transform(self, data):
+        transformed = super()._transform(data)
+        transformed_vals = transformed if transformed.ndim == 1 else transformed[:, 0]
+        self._validate_logit_inputs(transformed_vals)
+        logit_vals = logit(transformed_vals, self.min_value, self.max_value)
+        if transformed.ndim == 1:
+            return logit_vals
+        else:
+            transformed[:, 0] = logit_vals
+            return transformed
+
+    def _reverse_transform(self, data):
+        if not isinstance(data, np.ndarray):
+            data = data.to_numpy()
+
+        sampled_vals = data if data.ndim == 1 else data[:, 0]
+        reversed = sigmoid(sampled_vals, self.min_value, self.max_value)
```

Comment:
instead of `reversed` could we use `reversed_values` or `reversed_vals` since `reversed` is reserved word in python